client/components/domains/search-filters/tld-filter-bar.jsx

```diff
@@ -125,16 +127,17 @@ export class TldFilterBar extends Component {
 	}
 
 	renderPopoverButton() {
+		const { filters: { tlds = [] } = {}, translate } = this.props;
 		return (
 			<Button
 				className={ classNames( 'search-filters__popover-button', {
-					'is-active': this.props.filters.tlds.length > 0,
+					'is-active': tlds.length > 0,
 				} ) }
 				onClick={ this.togglePopover }
 				ref={ this.bindButton }
 				key="popover-button"
 			>
-				{ this.props.translate( 'More Extensions', {
+				{ translate( 'More Extensions', {
```

Comment:
:disappointed: 18 existing translations will be lost with this change.

Reply:
You're wrong, bot.